### PR TITLE
Services migration adaptations

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
looks like I forgot to create this PR
It looks big but it's mostly deleted files and function splits

### CODE REVIEW

File: octobot_interfaces/api/interfaces.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#  Drakkar-Software OctoBot-Interfaces
```
Comment (Herklos) [inline octobot_interfaces/api/interfaces.py]: Can you add some typing for params ?

File: octobot_interfaces/managers/interface_manager.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#  Drakkar-Software OctoBot-Interfaces
+#  Drakkar-Software OctoBot-Interfaces#  Drakkar-Software OctoBot-Interfaces
```
Comment (Herklos) [inline octobot_interfaces/managers/interface_manager.py]: Dupliace `#  Drakkar-Software OctoBot-Interfaces` should be removed ;)

File: octobot_interfaces/managers/interface_manager.py [hunk 3]
```diff
@@ -14,13 +14,15 @@
 #  You should have received a copy of the GNU Lesser General Public
 #  License along with this library.
 
-from flask import render_template
 
-from interfaces.web import server_instance, get_logs, flush_errors_count
+class InterfaceManager:
```
Comment (Herklos) [inline octobot_interfaces/managers/interface_manager.py]: I don't understand the need for a class here. Because all its methods are @static.

**Comment by Herklos** [comment]
I think it's relevant to not cythonize any file in this module. But I also think that we should add more typing to increase reliability (not provided by cythonization). What do you think ?

**Comment by GuillaumeDSM** [comment]
> I think it's relevant to not cythonize any file in this module. But I also think that we should add more typing to increase reliability (not provided by cythonization). What do you think ?

Agree, and I will add a return value to start_interfaces

**Comment by Herklos** [comment]
Perfect 👌

**Pull request merged**